test/blackbox-tests/test-cases/doc-browser.t/run.t

```diff
@@ -0,0 +1,12 @@
+This tests shows how to use the `dune ocaml doc` command to open the
+documentation index to a browser.
+  $ ls
+  dune
+  dune-project
+  foo.ml
+  spoof.desktop
+  xdg-open
+  $ PATH=.:$PATH 
+  $ export PATH
+  $ dune ocaml doc | grep -c "file:///.*/_build/.sandbox/.*/default/test/blackbox-tests/test-cases/doc-browser.t/_build/default/_doc/_html/index.html"
```

Comment:
```suggestion
  $ dune ocaml doc | sed -e 's/.*file:\/\/\([^ ]*\).*/\1/'
  $TESTCASE_ROOT/_build/default/_doc/_html/index.html
```
Here is a better way to scrub the output. cram tests usually detect and scrub the current directory but because we start with `file:///` it gets confused. Therefore we can manually remove it with sed.